meeting.js

```diff
@@ -0,0 +1,330 @@
+    joinMeetingButton.style.display = 'none';
+  } catch (error) {
+    console.error('Error joining meeting with media:', error);
+    throw error;
+  }
+}
+
+// 10) Join Meeting Listener and Calling Above Functions(6 to 9).
+joinMeetingButton.addEventListener('click', joinMeeting);
+async function joinMeeting() {
+  document.getElementById("joinMeetingButton").style.display = "none";
+  document.getElementById("localvideoimage").style.display = "none";
+  $('#joinMeetingModal').modal('show');
+  var element = document.querySelector('.container3');
+  element.classList.add('blur');
+  try {
+    await createMeeting();
+    setMediaListeners();
+    localStream = await getLocalStreams();
+    createdMeeting.setRemoteQualityLevel('HIGH');
+    await joinMeetingWithMedia(localStream);
+  } catch (error) {
+    console.error('Error joining meeting:', error);
+  }
+}
+
+// 11) Leaving Meeting Listener and Function.
+leaveMeetingButton.addEventListener('click', leaveMeeting);
+async function leaveMeeting() {
+  try {
+    const confirmLeave = window.confirm("Are you sure you want to leave the meeting?");
+    if(confirmLeave){
+      await createdMeeting.leave();
+      joinMeetingButton.style.display = 'block';
+      $('#joinMeetingModal').modal('hide');
+      videoButton.innerHTML = '<i class="fas fa-video icon green"></i>';
+      microphoneButton.innerHTML = '<i class="fas fa-microphone icon green"></i>';
+      microphoneButton.style.padding = "14px 21px";
+      bnrButton.classList.remove('selected');
+      vbgButton.classList.remove('selected');
+      createdMeeting=null;
+      localStream = null;
+      isMuted = true;
+      isVideoStarted = true;
+      vbgEffect = false;
+      
+    }
+  } catch (error) {
+    console.error('Error leaving meeting:', error);
+    $('#joinMeetingModal').modal('hide');
+
+  }
+}
+
+// 12) Start and Stop Button using toggleVideo Function.
+videoButton.addEventListener('click', toggleVideo);
+async function toggleVideo() {
+  if (!localStream || !localStream.camera || !localStream.camera.outputStream) {
+    console.error('No local video stream available.');
+    return;
+  }
+  // Stop Video
+  if (isVideoStarted) {
+    await createdMeeting.unpublishStreams([localStream.camera]);
+    localVideo.srcObject = null;
+    document.getElementById("localvideoimage").style.display = "block";
+    videoButton.innerHTML = '<i class="fas fa-video-slash icon red"></i>';
+    isVideoStarted = false;
+    vbgButton.classList.remove('selected');
+    if (vbgEffect && vbgEffect.isEnabled) {
+      await vbgEffect.disable();
+    }
+  } 
+  // Start Video
+  else {
+    const cameraStream = await webex.meetings.mediaHelpers.createCameraStream({ width: 640, height: 480 });
+    localStream.camera = cameraStream;
+    localVideo.srcObject = cameraStream.outputStream;
+    await createdMeeting.publishStreams({ camera: localStream.camera });
+    videoButton.innerHTML = '<i class="fas fa-video icon green"></i>';

```

Comment:
Same here. Applicable everywhere

Reply:
Have you explored the [javascript classList](https://www.w3schools.com/jsref/prop_element_classlist.asp) property?

Reply:
Yes I had Explored Class List Property And applied But it did not Works As Expected Compare to innerHtml